pkg/async-command/DependencyInjection/AsyncCommandExtension.php

```diff
@@ -12,13 +12,22 @@ class AsyncCommandExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         foreach ($configs['clients'] as $client) {
-            $id = sprintf('enqueue.async_command.%s.run_command_processor', $client);
+            // BC compatibility
+            if (!is_array($client)) {
+                $client = [
+                    'name' => $client,
+                    'prefix' => '',
+                    'timeout' => 60,
+                ];
+            }
+
+            $id = sprintf('enqueue.async_command.%s.run_command_processor', $client['name']);
             $container->register($id, RunCommandProcessor::class)
-                ->addArgument('%kernel.project_dir%')
+                ->addArgument('%kernel.project_dir%', $client['timeout'])
                 ->addTag('enqueue.processor', [
-                    'client' => $client,
-                    'command' => Commands::RUN_COMMAND,
-                    'queue' => Commands::RUN_COMMAND,
+                    'client' => $client['name'],
+                    'command' => $client['prefix'].Commands::RUN_COMMAND,
```

Comment:
IMO prefix is misleading, what about to allow configure a whole command and queue names?

Reply:
The default value could be the one from `Commands::RUN_COMMAND`

Reply:
@makasim  sounds like a great idea. I'm going to implement the fix